### PR TITLE
set proper bitwidth for verifreg command hamts

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: cmd/lotus-shed/verifreg.go [hunk 1]
```diff
@@ -268,13 +271,22 @@ var verifRegListClientsCmd = &cli.Command{
...
+			fmt.Printf("%s: %s %v\n", addr, dcap, []byte(k))
+
+			keys = append(keys, k)
 
 			return nil
 		}); err != nil {
 			return err
 		}
 
+		for _, k := range keys {
```
Comment (arajasek) [inline cmd/lotus-shed/verifreg.go:283]: Did you mean to add this (and if so, do you wanna add it to list-verifiers too)?

File: cmd/lotus-shed/verifreg.go [hunk 2]
```diff
@@ -268,13 +271,22 @@ var verifRegListClientsCmd = &cli.Command{
...
+			_, err := vh.FindRaw(ctx, k)
+			if err != nil {
+				fmt.Println("failed to find key: ", []byte(k), err)
+			}
+		}
+
```
Comment (magik6k) [inline cmd/lotus-shed/verifreg.go:289]:
```suggestion
```

File: cmd/lotus-shed/verifreg.go [hunk 3]
```diff
@@ -312,14 +324,15 @@ var verifRegCheckClientCmd = &cli.Command{
 			return err
 		}
 
-		vh, err := hamt.LoadNode(ctx, cst, st.VerifiedClients)
+		vh, err := hamt.LoadNode(ctx, cst, st.VerifiedClients, hamt.UseTreeBitWidth(5))
 		if err != nil {
 			return err
 		}
 
+		fmt.Println("key: ", caddr.Bytes())
```
Comment (magik6k) [inline cmd/lotus-shed/verifreg.go:332]:
```suggestion
```

**Comment by whyrusleeping** [comment]
@magik6k @arajasek good to go now

**Pull request merged**